### ISSUE TITLE
Documentation missing param descriptions

### ISSUE THREAD

**Issue by bbergstrom** [t7mp/t7mp#67]
Descriptions of what all the goal parameters do would be great.  I was banging my head because my integration tests weren't running.  I finally figured out that tomcatSetAwait was defaulting to true, which blocks maven from proceeded to my tests even when I was using run-forked.

Here is the integration test configuration that works for me using selenium:

``` xml
<profile>
            <id>it</id>
            <build>
                <plugins>
                    <plugin>
                        <artifactId>maven-javadoc-plugin</artifactId>
                        <configuration>
                            <!--Javadoc is noisy, expensive, and not needed during integration testing build.-->
                            <skip>true</skip>
                        </configuration>
                    </plugin>
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-failsafe-plugin</artifactId>
                        <version>2.6</version>
                        <configuration>
                            <forkMode>never</forkMode>
                        </configuration>
                        <executions>
                            <execution>
                                <id>integration-test</id>
                                <goals>
                                    <goal>integration-test</goal>
                                </goals>
                            </execution>
                            <execution>
                                <id>verify</id>
                                <goals>
                                    <goal>verify</goal>
                                </goals>
                            </execution>
                        </executions>
                    </plugin>

                    <plugin>
                        <groupId>com.googlecode.t7mp</groupId>
                        <artifactId>maven-t7-plugin</artifactId>
                        <version>0.9.10.M8</version>
                        <configuration>
                            <tomcatVersion>7.0.27</tomcatVersion>
                            <contextPath>/foo</contextPath>
                            <tomcatHttpPort>8080</tomcatHttpPort>
                            <tomcatSetAwait>false</tomcatSetAwait>
                            <systemProperties>
                                <profile>it</profile>
                            </systemProperties>
                        </configuration>
                        <executions>
                            <execution>
                                <id>start-container</id>
                                <phase>pre-integration-test</phase>
                                <goals>
                                    <goal>run-forked</goal>
                                </goals>
                            </execution>
                            <execution>
                                <id>stop-container</id>
                                <phase>post-integration-test</phase>
                                <goals>
                                    <goal>stop-forked</goal>
                                </goals>
                            </execution>
                        </executions>
                    </plugin>
                </plugins>
            </build>
        </profile>
```

**Comment by jbellmann** [comment]
Hi Brian,

you are right. I'm sorry. Thanks for posting your configuration for other users.

Cheers
Jörg

**Comment by bbergstrom** [comment]
Turns out doing run-forked/stop-forked maven was not able to actually stop tomcat and I had a runaway tomcat every time. :(

I changed to use run/stop and all is well. :)

**Comment by jbellmann** [comment]
Hi Brian,

I have updated the documentation about 'integration-testing' with the plugin [here](http://t7mp.github.com/how-to-run-integration-test.html).

And I have added an 'integration-testing'-module in our 'examples'-project [here](https://github.com/t7mp/examples/tree/master/integration-testing).

I will invest more time for documentation in the next days.

Thanks again for sharing your configuration.

Cheers
Jörg
